src/functions/postResult/framework/handler.ts

```diff
@@ -15,6 +15,12 @@ export async function handler(event: APIGatewayProxyEvent, fnCtx: Context): Prom
 
   let testResult: StandardCarTestCATBSchema;
 
+  let isPartialTestResult = false;
+  if (event.queryStringParameters && event.queryStringParameters['partial']
+    && event.queryStringParameters['partial'] === 'true') {
```

Comment:
not sure if it's absolutely needed but transforming query param to lower case before the comparison might help to avoid any case sensitive issues

Reply:
added `.toLowerCase()`